src/content/pages/thank-you.mdx

```diff
@@ -0,0 +1,273 @@
+
+- Mia Bajic
+- Marcin Wierzbanowski
+- Artur Czepiel
+- Ege Akman
+
+Big shoutout to **Daksh** for the website design, and to **Marcin Wierzbanowski** for rewriting the entire website and refactoring our codebase. 
+Another big thanks to **Marcin Wierzbanowski** for helping automating design assets creation.
+
+Thanks to **Panagiotis Kyrillos** for the EP2025 logo, based on **Adriana Sanchéz**’s astronomical clock design from 2024.
+
+## Sponsorship
+
+Conference income comes from ticket sales and sponsorships. The Sponsors Team contacts sponsors early, manages contracts, invoices, logistics, exhibits, and ensures all contract deliverables are fulfilled by other teams.
+
+The team:
+- Artur Czepiel
+- Iryna Kondrashchenko
+- Jake Balaš
+- Konstantin Ignatov
+- Kshitijaa Jaglan
+- Marcin Wierzbanowski
+- Mia Bajić
+- Oleh Kostromin
+- Raquel Dou
+
+## Financial Aid
+
+Speakers or attendees who need financial assistance to attend can apply for support. This team manages financial aid applications, selects recipients, handles communications, and ensures invoices are submitted:
+
+- Vicky Twomey-Lee
+- Doreen Peace Nangira Wanyama
+- Koti Vellanki
+- Shekhar Koirala
+- Artur Czepiel
+
+
+## Operations
+This is the largest team, divided into several smaller teams. You can meet them onsite or when contacting our helpdesk, staffed weekly by rotating team members.
+
+Operations Team:
+
+- Aneta Popelová
+- Anežka Müller
+- Aris Nivorlis
+- George Margaritis
+- George Zisopoulos
+- Jakub Červinka
+- Jake Balaš
+- John Robert
+- Kateřina Vaňková
+- Liberta Gani
+- Mahe Iram Khan
+- Martin Borus
+- Moisés Guimarães
+- Niklas Mertsch
+- Piotr Gnus
+- Saurav Pandey
+- Simon Peter Dagbui
+
+
+## Discord Team
+
+If you have a valid ticket, you have received access to our Discord server where you can chat with fellow participants, speakers, or organizers. This team manages the Discord server, ensuring only participants have access and proper permissions.
+
+Huge thanks to **Niklas Mertsch** for leading Discord and bot management!
+
+The work on maintaining and updating the discord bot can be found on the [EuroPython discord bot repo](https://github.com/EuroPython/discord).
+
+
+## A/V Team
+Although an external supplier handles streaming and recording, this team communicates with the venue, manages digital screens, and handles unforeseen technical issues. 
+
+Big thanks to **Raquel Dou** and **Piotr Gnus** for A/V coordination.
+
+
+## Onsite Volunteers Team
+
+Once you enter the conference, the first what you see are people in yellow t-shirts. They're our volunteers, who assist attendees, chair sessions, and hand out materials. 
+The onsite volunteers sub-team manages volunteer recruitment and scheduling:
+- xxx
+
+
+# Onsite & Remote Volunteers
+Volunteers visible at the conference in yellow shirts, and those supporting remotely:
+
+## Onsite Volunteers
+- xxx 
+
```

Comment:
Special thanks to the volunteers that made conference a suitable place for a couple of visual impaired participants/speakers during the whole event:

- Raquel
- Konstantin
- Daniel Clauser
- Vega Čerkassova
- Aleksandr Salynskii
- Evgeniya Volkova

Reply:
Remember to ask the most fantastic Jake! <3